github-upload-action-secrets

```diff
@@ -88,6 +88,8 @@ def main():
                       help="Upload given directory with one file per secret")
     megr.add_argument('--ssh-keygen', metavar="SECRET_NAME",
                       help="Generate an SSH key, upload private, public to stdout")
+    parser.add_argument('--deploy-to', metavar="OWNER/REPO",
```

Comment:
At first sight I thought "wait, this wrong", and only then I saw that this is to `parser`, not `megr`. Could you move this up, or leave a new line or so?

Reply:
Sure.